frontend/src/scss/PrimeStyles.scss

```diff
@@ -789,3 +805,47 @@ $results-dropdown-spacing: #{units(4)} - #{units(2)} - 22px - #{units(4)}; // he
   padding: units(2.5) units(3);
   position: relative;
 }
+
+.modal__footer--sticky {
+  @include at-media(mobile) {
+    z-index: 1;
+    position: absolute;
+    width: 100%;
+    right: 0;
+    bottom: 0;
+    padding: units(2.5);
+    box-shadow: 0 units(-0.5) units(1) 0 rgba(0, 0, 0, 0.1);
+    background: color("white");
+  }
+
+  @include at-media(desktop) {
+    position: fixed;
+    width: unset;
+    right: unset;
+    padding: 0;
+    bottom: units(7);
+  }
+}
+
+.modal__submit--container {
```

Comment:
unless this is something we want to generalize to other modals, use `aoe-modal` as your "block" in BEM. `container ` is not a modifier of `submit`, it's either another element or a description of submit, so just keep to `submit` or `submit-container`